packages/shared/routes/dashboard/governance/views/GovernanceEventDetails.svelte

```diff
@@ -40,6 +60,17 @@
         }
     }
 
+    $: results = event?.status?.questions?.[0]?.answers?.filter(
+        (answer) => answer?.value !== 0 && answer?.value !== 255
+    )
+    $: totalVotesResult = results?.reduce((acc, val) => acc + val?.accumulated, 0)
```

Comment:
`totalVotesResult` and `totalVotes` are 2 variables that are called quite similar, would it be possible to maybe rename them so its more clear what they mean?